docker-compose-bblab.yml

```diff
@@ -51,8 +51,8 @@ services:
     #          in this container. Apache will set the URL from 
     #          BBLAB_WEB_ADDRESS (use "localhost" or your DNS)
     #          and Django will accept requests to that address
-    # ports:
-    #   - "80:80/tcp"
+    ports:
+      - "80:80/tcp"
```

Comment:
Does this expose port 80 on the live server?

I think this can be useful in a dev environment, but maybe not on the live server.

Reply:
Yes, this is an artifact from my development environment. Reverted the change.